static/templates/stream_types.hbs

```diff
@@ -37,9 +37,9 @@
         {{> settings/upgrade_tip_widget}}
 
         <div class="input-group inline-block message-retention-setting-group">
-            <label for="stream_message_retention_setting" class="dropdown-title">{{t "Message retention period" }}
+            <div class="dropdown-title">{{t "Message retention period" }}
```

Comment:
Semantically, this should probably be an h3 or something like what we use in settings.